controller/array_action/array_mediator_ds8k.py

```diff
@@ -95,6 +95,13 @@ def shorten_volume_name(name, prefix):
         return (prefix + settings.NAME_PREFIX_SEPARATOR + hashed)[:MAX_VOLUME_LENGTH]
 
 
+def shorten_snapshot_name(name):
```

Comment:
I think we can remove all name `shorten` logic in ds8k, and simply change `max_volume_name_length` from 63 to 16, since we now have generic logic: https://github.com/IBM/ibm-block-csi-driver/pull/180